app/views/datafiles/_dataset_field.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="form-group<%= " error" if dataset.errors.include?(field[:name])%>" id="<%= field[:name] %>-form-group">
```

Comment:
this fill will probably conflict with #116 .
Also, I'm a bit confused by this file as it seems to be a generic partial for forms controls, which is great but other templates aren't using it systematically.

(which reminds me I'm considering using https://github.com/ministryofjustice/govuk_elements_form_builder as a possible improvement to govuk forms)